development/fake-cht/docker-compose.fake-cht.yml

```diff
@@ -17,7 +17,7 @@ services:
              npm start"
 
   postgres:
-    image: postgres:latest
+    image: postgres:15
```

Comment:
What do you think about leaving this as `latest`?  At this point we are setting a definite _minimum limit_ for which version of postgres we support, but we are not yet tracking any max version.  From a testing perspective it might be nice to have a heads-up that a new version of postgres is incompatible. ....

Reply:
I did think about leaving the `latest` tag for local development images and I don't have a strong opinion for either solution

I'm merging the PRs as they are and we can keep the discussion going